.evergreen/config.yml

```diff
@@ -859,14 +859,18 @@ axes:
         variables:
            TOX_ENV: "tornado6-py310"
            PYTHON_BINARY: "/opt/python/3.10/bin/python3"
+      - id: "tornado6-py311"
+        variables:
+           TOX_ENV: "tornado6-py311"
+           PYTHON_BINARY: "/opt/python/3.11/bin/python3"
       - id: "tornado_git-py38"
         variables:
            TOX_ENV: "tornado_git-py38"
            PYTHON_BINARY: "/opt/python/3.8/bin/python3"
-      - id: "asyncio-pypy37"
+      - id: "asyncio-pypy38"
```

Comment:
Is there another platform we can test with pypy 3.7?

Reply:
Found a workaround, by adding an upper limit for that combination.